src/net/sf/freecol/client/gui/GUI.java

```diff
@@ -1570,6 +1570,17 @@ public void clearGotoPath() {}
      */
     public void performGoto(Tile tile) {}
 
+    /**
+     * Check if the user has  GoTo mode enabled.
+     *
+     * Called from {@link CanvasMouseListener}.
+     *
+     * @return True if the user has toggled GoTo mode.
+     */
+    public boolean isGotoStarted() {
+        return false;
```

Comment:
I'm a bit confused by this code block. The comment says it will return true if the user has toggled GoTo mode, but this method returns false.

Reply:
Its the base class, since it required a method body I thought it would be reasonable to default it to false. Check SwingGUI.java for the override. Actually speaking of, let me patch it with an Override annotation